complex_envs/rollout.py

```diff
@@ -16,8 +16,6 @@
 from stable_baselines.common.vec_env import DummyVecEnv
 from stable_baselines import PPO2
 
-#currently this just loads the simple corridor environment, more will be added later
-from simple_environment import SimpleCorridor
 from active_damping_env import VibratingBridge
 
 #other utilities
```

Comment:
commenting practice (here and elsewhere) is # Starting with a space and capital letter

Reply:
Ok, I will clean this up over the weekend.